client/package.json

```diff
@@ -48,6 +48,7 @@
   },
   "devDependencies": {
     "@craco/craco": "^7.1.0",
+    "@types/jest": "^30.0.0",
```

Comment:
We are not using Jest 30 though, we are using Jest 27 for frontend according to the lockfile